packages/babel-core/test/config-chain.js

```diff
@@ -384,7 +383,7 @@ describe("buildConfigChain", function() {
           comments: true,
         });
 
-        assert.equal(opts.comments, undefined);
```

Comment:
`toBe(undefined)` could be `toBeUndefined()`
also in other places

Reply:
Changed. (Y)